mopidy/core/mixer.py

```diff
@@ -37,8 +58,12 @@ def set_volume(self, volume):
 
         if self._mixer is None:
             return False
-        else:
-            return self._mixer.set_volume(volume).get()
+
+        with _mixer_error_handling(self._mixer):
+            # TODO: log non-bool return values?
```

Comment:
Do we know of any mixers not returning a bool? The mixer API docs are quite clear on the return type, so any mixers not returning a bool falls clearly under they having a bug, and not us breaking the API.